src/workloads/scale/ScanWithLongLived.yml

```diff
@@ -0,0 +1,82 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
+
+GlobalDefaults:
+  TrackProportion: &TrackProportion 0
+
+  LongLivedDB: &LongLivedDB longlived # name of the long lived database
+  LongLivedCollectionCount: &LongLivedCollectionCount 1000
+  LongLivedDocumentCount: &LongLivedDocumentCount 1000
+
+  ScannerHotDB: &ScannerHotDB rolling
+
+  LongLivedWrites: &LongLivedWrites 600 per 1 second # x5 threads
+  LongLivedReads: &LongLivedReads 20 per 1 second    # x10 threads
+
+  binomial_10k_int: &binomial_10k_int {^RandomInt: {distribution: binomial, t: 1000, p: 0.5}}
```

Comment:
I wonder if `t: 1000` should be `t: *LongLivedDocumentCount`

Reply:
These two values are related (e.g. it doesn't make sense to make the doc count much smaller than the key-range) but I don't think it's clear that they will remain equal to each other.